lib/OpenQA/Worker/Jobs.pm

```diff
@@ -530,7 +530,9 @@ sub upload_status(;$) {
         $status->{log}             = log_snippet("$pooldir/autoinst-log.txt",   \$log_offset);
         $status->{serial_log}      = log_snippet("$pooldir/serial0",            \$serial_offset);
         $status->{serial_terminal} = log_snippet("$pooldir/virtio_console.log", \$serial_terminal_offset);
-        print STDERR $status->{serial_terminal}->{data};
+        if ($status->{serial_terminal}->{data}) {
+            print STDERR $status->{serial_terminal}->{data};
```

Comment:
I really don't think we want the contents of data to be printed to STDERR except for debugging maybe.